tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_custom_CTabFolder.java

```diff
@@ -468,6 +468,20 @@ public void test_iconWrappedOnNextLine() {
 	}
 }
 
+@Test
+public void test_selectedImageVisible() {
+	createTabFolder(null);
+
+	ctabFolder.setSelectedImageVisible(true);
+	assertTrue( ctabFolder.getSelectedImageVisible() );
+
+	ctabFolder.setSelectedImageVisible(false);
+	assertFalse( ctabFolder.getSelectedImageVisible());
+
```

Comment:
Please correct formatting here. There are strange whitespaces that will be corrected with auto-formatting.